compiler/common-artifacts/CMakeLists.txt

```diff
@@ -33,13 +33,18 @@ set(REQUIREMENTS_FILE "requirements.txt")
 set(REQUIREMENTS_OVERLAY_PATH_TF_2_6_0 "${VIRTUALENV_OVERLAY_TF_2_6_0}/${REQUIREMENTS_FILE}")
 set(REQUIREMENTS_OVERLAY_PATH_TF_2_8_0 "${VIRTUALENV_OVERLAY_TF_2_8_0}/${REQUIREMENTS_FILE}")
 
+set(PYTHON_OVERLAY python3)
```

Comment:
```cmake
find_package(PythonInterp 3.8 QUIET)
find_package(PythonLibs 3.8 QUIET)
```
Is it okay not to change above line?

Reply:
Above line needs to find python version >= 3.8.